.github/workflows/example-scan-image.yaml

```diff
@@ -0,0 +1,23 @@
+name: Kubescape scanning for image vulnerabilities
+on: [push, pull_request]
+jobs:
+  kubescape-scan-image:
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+    steps:
+    - uses: actions/checkout@v3
+    - uses: kubescape/github-action@feat-image-scanning
+      with:
+        image: "nginx"
+        format: "pretty-printer"
+        severityThreshold: "critical"
```

Comment:
chore: we should leave the `severityThreshold` unset but mention it in the commented examples below

If we show less knobs to turn from the get-go, a user should be able to gain value faster from our Github Action. Moreover, our default severity threshold is at High, and it provides more value than gating only at Critical.